colossalai/inference/utils.py

```diff
@@ -102,3 +104,11 @@ def get_model_size(model: nn.Module):
     for key, param in model.named_parameters():
         total_size += param.element_size() * param.numel()
     return total_size / (1024**3)
+
+
+def find_available_ports(num: int):
+    try:
+        free_ports = [free_port() for i in range(num)]
+    except OSError:
+        return
```

Comment:
Lack of handling the exception (for example, looking for another free ports or something) and warnings or error reporting

Reply:
Thx! Fixed!